cognite/neat/graph/extractors/_iodd.py

```diff
@@ -368,9 +372,9 @@ def _extract_enhanced_ts_information(self, json_file_path: Path):
             self.root, "ProcessDataIn", ignore_namespace=True, include_nested_children=True
         ):
             for process_data_element in process_data_in:
-                if id := process_data_element.attrib.get("id"):
+                if process_data_element.attrib.get("id"):
                     device_id_str = remove_namespace_from_uri(self.device_id)
-                    process_data_in_id = f"{device_id_str}.{id}"
+                    process_data_in_id = f"{device_id_str}.{P_DATA_IN_ID_OVERRIDE}"
                     if record_items := get_children(
```

Comment:
I mentioned above that this will not produce unique IDs, so for you to complete this you will need to read `StdVariableRef` from `IODD-StandardDefinitions1.1.xml`.